src/clients/api/mutations/useEnterMarkets.ts

```diff
@@ -0,0 +1,29 @@
+import { useMutation, MutationObserverOptions } from 'react-query';
+
+import { enterMarkets, IEnterMarketsInput, EnterMarketsOutput } from 'clients/api';
+import FunctionKey from 'constants/functionKey';
+import { useComptroller } from 'hooks/useContract';
+
+const useEnterMarkets = (
+  options?: MutationObserverOptions<
+    EnterMarketsOutput,
+    // @TODO: use custom error type (see https://app.clickup.com/t/2rvwhnt)
```

Comment:
I think if we are going to leave TODOs having a ticket and pointing to it is the right way to do it